<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="Rachunek_dec" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#Rachunek_dec{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(179, 204, 117, 255), stop:1 rgba(255, 255, 255, 255))};</string>
   </property>
   <widget class="QLabel" name="tekst_glowny">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>28</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;MS Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>RACHUNEK DECYBELOWY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oblicz_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>620</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Oblicz</string>
    </property>
   </widget>
   <widget class="QLabel" name="tekst_wyniki">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 15pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>WYNIK</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Wróć do głównego menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>620</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="tekst_podaj_dane">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 15pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>PODAJ DANE WEJŚCIOWE</string>
    </property>
   </widget>
   <widget class="QLabel" name="tekst_wybierz_rodzaj">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 15pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>WYBIERZ RODZAJ KONWERSJI</string>
    </property>
   </widget>
   <widget class="QComboBox" name="wybor_konwersji">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>dBW to dBm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dBm to dBW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dBW to W</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dBm to W</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>W to dBm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dB to Ratio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ratio to dB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Loss to dB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltage to dB</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="pierwsza_dana">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="druga_dana">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>400</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="jednostka_danych1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>360</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="jednostka_danych2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>400</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="jednostka_wyniku">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="wynik">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>490</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
